### PR TITLE
Fix remove team member issue

### LINKED ISSUE #3410: Removing a user from an organization
- Gitea version (or commit ref): 1.3.2
- Git version: 2.11.0
- Operating system: Debian Stretch
- Database:
  - [X] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant

## Description
A user is not removed from an organization's member list when removed from (the last team). The user has no access to the repositories but she can still enumerate all team members of the organization.

### Steps to reproduce
* Create a new organization
* Add a private repository to the organization (owner team)
* Create a new user: user1
* Add a new team: team1 (read perm with access to the private repository)
* Add user1 to team1
* User1 can clone the repository and inspect it in the UI: OK
* Remove user1 from team1: user1 is no longer a member of any team in the organization. She is, however, still listed under the organization's members list (People).

Therefore, to fully remove a user from an organization, one has to remove it from all teams and remove it from the members list. I expected that the user is automatically removed from the members list as soon as she is no longer part of any team.

### PR DESCRIPTION
**Pull request by xwjdsh**
Fixes #3410

### CODE REVIEW

File: models/org.go [hunk 1]
```diff
@@ -523,6 +516,16 @@ func RemoveOrgUser(orgID, userID int64) error {
 	return sess.Commit()
```
Comment (lunny) [inline models/org.go:436]: Put `sess.Commit()` out of the function is a good habit I think.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by xwjdsh** [comment]
@lunny You are right, changed.

**Review by lunny — APPROVED** [review]

File: models/org_team.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 
 	"code.gitea.io/gitea/modules/log"
+	"github.com/go-xorm/xorm"
```
Comment (appleboy) [inline models/org_team.go]: Add an empty line between local and external package.

**Review by appleboy — CHANGES_REQUESTED** [review]
coding style for import package

**Comment by xwjdsh** [comment]
@appleboy Updated.

**Review by sapk — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Comment by lafriks** [comment]
@xwjdsh can you backport this to `release/v1.4` branch?

**Pull request merged**